### ISSUE TITLE
The confirm message of azure-spring:deploy is not complete

### ISSUE THREAD

**Issue by ZhengJun003** [microsoft/azure-maven-plugins#778]
### Plugin name and version
```
azure spring maven plugin
0.0.1-SNAPSHOT
```
### Plugin configuration in your `pom.xml`
```
<plugin>
     <groupId>com.microsoft.azure</groupId>
     <artifactId>azure-spring-maven-plugin</artifactId>
     <version>0.0.1-SNAPSHOT</version>
</plugin> 
```

### Actual behavior
![image](https://user-images.githubusercontent.com/41051257/63999859-0a6bfd80-cb37-11e9-9261-8148a6e2021a.png)

### Steps to reproduce the problem

1. Prepare a jar maven project and add the plugin at project/build/plugins
2. Run mvn azure-spring:deploy -Dprompt after config

**Comment by Flanker32** [comment]
Fixed in #781

**Comment by ZhengJun003** [comment]
Fixed
Test in the branch feature-spring
